src/main/configurations/Translate/DeploymentSpecifics.properties

```diff
@@ -95,7 +95,7 @@ ISO639_Codes_path=${configurations.directory}/Translate/ISO639.xml
 
 creeerZaak.timeout=20000
 
-Prefixparam=1900
+zaak-identificatie-template=ZK{datetime:[Y0001]}-{id:D5}
```

Comment:
The same mechanism should also be implemented for document identificatie with its own template in the DeploymentSpecifics.